getting-started/getting-started.md

```diff
@@ -11,75 +11,71 @@ position: 0
 # Getting Started with Telerik Document Processing
 
 
-Telerik Document Processing includes a set of cross-platform libraries that let you import and export content between different formats and work with archive files. The needed assemblies come with the installation (automated and manual) of several of the Telerik suites and in the internal builds archive where the latter is available. More information about the suites that include Telerik Document Processing and how to install them is available in the [Installing on Your Computer article]({%slug installation-installing-on-your-computer%}). 
+Telerik Document Processing includes a set of cross-platform libraries that let you import and export content between different formats and work with archive files. The needed assemblies come with the installation files of several Telerik suites. They are available also in the internal builds of the same Telerik suites. 
 
->The libraries support .NET 4 and later. .NET Standard-compatible binaries are available as well. The assemblies for .NET Standard don't include 'Windows' in their names.
-
-* **Telerik.Windows.Documents.Core.dll**: The main assembly from the Telerik document processing libraries. You always need it when you intend to use the library.
-
-* **Telerik.Windows.Documents.Flow.dll**: Part of the words (text) processing library. It is needed for processing HTML, DOCX, RTF and TXT documents.
+More information about the suites that include Telerik Document Processing and how to install them is available in the [Installing on Your Computer]({%slug installation-installing-on-your-computer%}) article.
 
-* **Telerik.Windows.Documents.Flow.FormatProviders.Pdf**: Allows exporting of flow documents (DOCX, RTF) to PDF.
-
-* **Telerik.Documents.SpreadsheetStreaming.dll**: The main assembly for generating huge spreadsheet files using minimum resources.
-
-* **Telerik.Windows.Documents.Spreadsheet.dll**: The main assembly for the spreadsheet processing. It is needed for processing XLSX, CSV, TXT documents.
+>The libraries support .NET 4 and later. .NET Standard-compatible binaries are available as well. The assemblies for .NET Standard don't include 'Windows' in their names.
 
-* **Telerik.Windows.Documents.Spreadsheet.FormatProviders.OpenXml.dll**: This assembly is needed for processing XLSX documents.
- 
-* **Telerik.Windows.Documents.Spreadsheet.FormatProviders.Xls.dll**: This assembly is needed for processing XLS documents.
+The following table lists all Telerik Document Processing assemblies and their roles:
```

Comment:
Such long lists are very hard to follow. In general, avoid lists that have more than 8 items. That's why I suggest using a table here.